server/pkg/verror/verror.go

```diff
@@ -0,0 +1,77 @@
+package verror
+
+import (
+	"strings"
+
+	"github.com/reearth/reearth/server/pkg/i18n"
+	"github.com/reearth/reearth/server/pkg/i18n/message"
+	"golang.org/x/text/language"
+)
+
+// VError represents an error with a code identifier.
+type VError struct {
+	Key          message.ErrKey
+	ErrMsg       map[language.Tag]message.ErrorMessage
+	TemplateData map[language.Tag]map[string]interface{}
+	Err          error
+}
+
+// GetErrCode returns the error code.
```

Comment:
We don't need long comments for functions since they already makes sense, Please comments in-case something complex has been implemented or you require the devs to be extra cautious about something.